Dockerfile

```diff
@@ -2,11 +2,27 @@ FROM python:3.6-jessie
 
 RUN useradd -ms /bin/bash ark
 
+ADD . /usr/src/code
+WORKDIR /usr/src/code
```

Comment:
These two lines are duplicated with the line 32 and 33. Please remove these and keep the ones on lines 32, 33. Reason being that if you put this before apt-get and some others, docker will not cache them as this lines will always be detected as a "change", causing docker build to take ages.